src/main/java/robosky/ether/mixin/PlayerMixin.java

```diff
@@ -65,8 +56,10 @@ private void updateParachute(CallbackInfo info) {
     }
 
     @Inject(method = "tickMovement", at = @At("TAIL"))
-    private void flyIntoUplands(CallbackInfo info) {
-        if (this.dimension == DimensionType.OVERWORLD && this.y >= 300.0) {
+    private void onTickMovement(CallbackInfo info) {
+        if (this.dimension == MixinHack.HOOKS.getDimensionType() && this.y <= -60) {
```

Comment:
Players start taking damage at `y = -40` so players in survival will take damage before getting teleported with this check

Reply:
Actually players start taking damage at y=-64. I tested in survival by falling off the uplands

Reply:
Huh, cool. Works for me then